app/models/cohort_columns/days_homeless_plus_overrides.rb

```diff
@@ -0,0 +1,21 @@
+###
+# Copyright 2016 - 2019 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+module CohortColumns
+  class DaysHomelessPlusOverrides < ReadOnly
+    attribute :column, String, lazy: true, default: :days_homeless_plus_overrides
+    attribute :translation_key, String, lazy: true, default: 'Days Homeless Plus Overrides*'
+    attribute :title, String, lazy: true, default: -> (model, attr) { _(model.translation_key)}
+
+    def description
+      'Days homeless + verified additional days'
+    end
+
+    def value(cohort_client) # OK
+      cohort_client.days_homeless_plus_overrides
```

Comment:
This should be consistent for the entire client (so we can pull from `client.processed_service_history&.days_homeless_plus_overrides`)